src/libopensc/card-idprime.c

```diff
@@ -156,13 +438,19 @@ static int idprime_select_index(sc_card_t *card)
 	return r;
 }
 
-static int idprime_process_index(sc_card_t *card, idprime_private_data_t *priv, int length)
+static int idprime_process_index(sc_card_t *card, idprime_private_data_t *priv, int length, idprime_container_t *containers, idprime_keyref_t *keyrefmap)
```

Comment:
can we store these in `idprime_private_data_t` instead of passing them as additional arguments?

Reply:
fixed